server/webapp/WEB-INF/rails.new/app/helpers/pipelines_helper.rb

```diff
@@ -30,7 +30,7 @@ def run_stage_label stage
 
   def trigger_message(actual_date_time, pim)
     if pim.class != com.thoughtworks.go.presentation.pipelinehistory.PreparingToScheduleInstance
-      "(Triggered&nbsp;#{l.string('BY')}&nbsp;<span class='who'>#{h pim.getApprovedBy()}</span>&nbsp;<span class='time'></span><input type='hidden' value='#{actual_date_time}'/>)"
+      "(Triggered&nbsp;#{l.string('BY')}&nbsp;<span class='who'>#{pim.getApprovedBy()}</span>&nbsp;on&nbsp;<span class='time'>#{actual_date_time.to_long_display_date_time()}</span><input type='hidden' value='#{actual_date_time}'/>)".html_safe
```

Comment:
Can we kill the parentheses around the text — it just seems to add to clutter.

Also - `html_safe` feels dangerous because we're rendering something we don't control (the login name). Can we instead use [`content_tag`](http://apidock.com/rails/ActionView/Helpers/TagHelper/content_tag) instead